### ISSUE TITLE
[Feature Request] Custom keyboard shortcuts to toggle global hotkey

### ISSUE THREAD

**Issue by Mj65536** [AlexanderPro/SmartContextMenu#28]
I want to quickly pause/resume the program without manually click taskbar to quit it or relaunch it.

**Comment by AlexanderPro** [comment]
Hi @Mj65536,

If you need to suspend (pause) your process you can use "System Tray | Suspend And Minimize To Tray" menu item.

**Comment by Mj65536** [comment]
> Hi @Mj65536,
> 
> If you need to suspend (pause) your process you can use "System Tray | Suspend And Minimize To Tray" menu item.

Sorry, I mean SmartContextMenu itself, is there any keyboard shortcuts or command to temporarily disable SCM's global hotkey without using mouse?

Most of the time SCM's global hotkey (e.g. Ctrl + RMB) will conflict with my game's hotkey, I have to quit SCM, then relaunch it while I am not playing games.

**Comment by AlexanderPro** [comment]
If I add a new menu item "Pause/Resume" to the system tray, without hotkeys, will it be convenient for you?

**Comment by Mj65536** [comment]
> If I add a new menu item "Pause/Resume" to the system tray, without hotkeys, will it be convenient for you?
Yeah it's okay😀,  but if you provide a command like the picture below,
![Snipaste_2024-11-28_21-29-39](https://github.com/user-attachments/assets/e9c87001-682d-443b-bd53-24fc70ab5716)

then I can define a custom keymap to toggle SCM's global hotkey, which is much more convenient for me ! 😍
